x/gamm/keeper/grpc_query.go

```diff
@@ -184,6 +184,8 @@ func (k Keeper) SpotPrice(ctx context.Context, req *types.QuerySpotPriceRequest)
 		return nil, status.Error(codes.InvalidArgument, "invalid denom")
 	}
 
+	// XXX: return error if inactive
```

Comment:
I think it will be to return pricing even though not started so that users can get the price if it starts right now.

Reply:
Agreed, it looks to me like that is what it will do, and that is the correct behavior.